src/main/pages/che-7/end-user-guide/assembly_adding-tools-to-che-after-creating-a-workspace.adoc

```diff
@@ -22,14 +22,14 @@ When installed in the workspace, {prod-short} plug-ins bring new capabilities to
 * Communicating with multiple databases directly from the IDE.
 * Enhanced code navigation, auto-completion and error highlighting.
 
-This chapter provides basic information about {prod-short} plug-in installation, enabling, and use in {prod-short) workspaces.
+This chapter provides basic information about Che Plugin installation, enabling, and use in {prod-short} workspaces.
```

Comment:
Would it be better to have `{prod-short} Plugin` here?

Reply:
It is indeed quite difficult to do the automated processing of all the  Plugin/plug-in occurrences.

- On one hand we have the generic family for all plug-ins that we should write `{prod-short} plug-in` and `{prod-short} plug-ins`.
- On the other hand we have a specific `type` of plug-in which is called `Che Plugin`  in the configuration files, and in that case attribute substitution is not an option.

Reply:
```suggestion
This chapter provides basic information about {prod-short} plug-in installation, enabling, and use in {prod-short} workspaces.
```